chunky/src/java/se/llbit/chunky/renderer/projection/OmniDirectionalStereoProjector.java

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2016 Chunky contributors
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.llbit.chunky.renderer.projection;
+
+import org.apache.commons.math3.util.FastMath;
+import se.llbit.math.Vector3;
+
+import java.util.Random;
+
+/**
+ * A projector for Omni-Directional Stereo (ODS) images. x is mapped to yaw, y is mapped to
```

Comment:
The "x is mapped to yaw, y is mapped to pitch" part is fairly non-descriptive. The comment is from the equirectangular projector which maps linearly between image coordinates and yaw/pitch. In the ODS projector the mapping is a bit more complicated so the comment should just be removed.

I didn't think about this before I merged your commits so I will update the comment myself. Just wanted to add this comment here for completeness sake.